### DISCUSSION TITLE
Impact of creating the full theme using createTheme and then defining it as a PartialTheme in public API

### DISCUSSION THREAD

**Discussion by AmyL219** [microsoft/fluentui#31755] [Q&A] [answered]
Our ACS Web library encountered an issue where the dark theme was not working as expected due to the absence of the isInverted property. To resolve this, we used the createTheme API to generate a full theme, ensuring the isInverted property was correctly set. 
[[A11y][Theme]Mark color inverted inside Dark Partial theme](https://github.com/Azure/communication-ui-library/pull/4736)
My question is: could creating the full theme using API `createTheme` and then defining it as a `PartialTheme` cause any potential issues for customers of the darkTheme when they want to use or override the darkTheme colors?

**Comment by khmakoto** [comment, marked as answer]
Creating a full `Theme` and passing it on to something that expects a `PartialTheme` should not create any problems. Nothing exists in `PartialTheme` that is required and doesn't exist also in the `Theme` definition.

**Reply by AmyL219** [reply]
Cool. Thanks
